observability/README.adoc

```diff
@@ -63,6 +63,26 @@ You can also directly leverage MicroProfile Health APIs to create checks. Class
 
 The tracing configuration for the application can be found within `application.properties`.
 
+Default configuration uses OTLP exporter, but it can be easily switched to Jaeger exporter with applying this change in `application.properties`:
```

Comment:
```suggestion
The default configuration uses the OTLP exporter, but it can be easily switched to the Jaeger exporter by applying this change in `application.properties`:
```

Reply:
Fixed, thanks @jamesnetherton